terraform/main.tf

```
@@ -0,0 +1,8 @@
+resource "null_resource" "nixos-remote" {
+  provisioner "local-exec" {
+    environment = {
+      SSH_PRIVATE_KEY = var.ssh_private_key
+    }
+    command = "${path.module}/../nixos-remote --store-paths ${var.nixos_partitioner} ${var.nixos_system} ${var.target_user}@${var.target_host} --debug"
```

Do we want `--debug` by default?
```suggestion
    command = "${path.module}/../nixos-remote --store-paths ${var.nixos_partitioner} ${var.nixos_system} ${var.target_user}@${var.target_host} --debug"
```

---

probably not, but we want it for debugging a WIP branch :)